js-intro.md

```diff
@@ -5,68 +5,72 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is a function? Why would you use one?
 
-  Your answer:
+  Your answer: To make some something work
 
-  Researched answer:
+  Researched answer: to perfom a task or calculate a value
 
 
 
 2. What is the difference between map and filter?
 
-  Your answer:
+  Your answer: map - tracking an element 
+  filter- testing something and provides you the correct answer
 
-  Researched answer:
+  Researched answer: a map is a collection of elements where each element is stored as a Key, value pair
+   a filter is a method creates an array filled with all array elements that pass a test
 
 
 
 3. What is the difference between console.log() and return?
 
-  Your answer:
+  Your answer: a console.log shows the user a message 
```

Comment:
`console.log()` shows the _developer_ a message. Used for de-bugging and testing. The user wouldn't see a `console.log()` unless they had Chrome developer tools open in their browser. This is why we end functions with a return instead.